decision_service/dsclientlib/Makefile

```diff
@@ -0,0 +1,28 @@
+TARGET = dsclientlib
+LIBS = -lm
+CXX = g++
+CXXFLAGS = -g -Wall -std=c++11
```

Comment:
>LAGS = -g -Wall -std=c++11 [](start = 4, length = 26)

can you re-use the Makefile infrastructure from VW?

Reply:
upgrade root Makefile to c++11

---
In reply to: [186742232](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186742232) [](ancestors = 186742232)

Reply:
Done

---
In reply to: [187632446](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r187632446) [](ancestors = 187632446,186742232)